backend_py/primary/primary/persistence/session_store/session_store.py

```diff
@@ -0,0 +1,240 @@
+from typing import List, Optional, Tuple
+from datetime import datetime, timezone
+from nanoid import generate
+
+from primary.persistence._utils import hash_session_content_string
+from primary.services.service_exceptions import Service, ServiceRequestError
+from primary.persistence.cosmosdb.cosmos_container import CosmosContainer
+from primary.persistence.cosmosdb.query_collation_options import Filter, QueryCollationOptions, SortDirection
+from primary.persistence.session_store.types import SessionSortBy
+from primary.persistence.cosmosdb.exceptions import DatabaseAccessError
+from primary.persistence.cosmosdb.error_converter import raise_service_error_from_database_access
+
+from .documents import SessionDocument, SessionMetadata
+
+
+class SessionStore:
+    """
+    A simple data store for session documents with CRUD operations.
+    Supports pagination, sorting, filtering, and limits.
+    """
+
+    CONTAINER_NAME = "sessions"
+    DATABASE_NAME = "persistence"
```

Comment:
These constants are only used internally in the class.
I suggest moving them to file/module scope as _DATABASE_NAME and _CONTAINER_NAME